data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/actions/HistogramAggregateAction.java

```diff
@@ -137,16 +138,29 @@ public AggregateActionResponse handleEvent(final Event event, final AggregateAct
             return AggregateActionResponse.nullEventResponse();
         }
         double doubleValue = convertToDouble(value);
-        
+
         int idx = Arrays.binarySearch(this.buckets, doubleValue);
         if (idx < 0) {
             idx = -idx-2;
         }
+        Instant eventTime = Instant.now();
+        Instant eventStartTime = Instant.now();
```

Comment:
Let's set these to the exact same value instead of calling `Instant.now()` multiple times.

```
Instant eventStartTime = eventTime;
Instant eventEndTime = eventTime;
```